### PR TITLE
More builtins

### PR DESCRIPTION
**Pull request by sbarzowski**

### CODE REVIEW

**Comment by sbarzowski** [comment]
@sparkprime @lwander

**Comment by sbarzowski** [comment]
hmm... coverage decreased, but this stuff is tested in "external" tests... hmmm... Maybe I should add individual single-line tests for each of these functions...

File: builtins.go [hunk 1]
```diff
@@ -248,8 +249,79 @@ func builtinType(e *evaluator, xp potentialValue) (value, error) {
...
+	if math.IsNaN(x) {
+		return nil, e.Error("Not a number")
+	}
+	if math.IsInf(x, 0) {
+		return nil, e.Error("Overflow")
+	}
+	return makeValueNumber(x), nil
+}
+
+// TODO(sbarzowski) perhaps it is too magical for Go style
```
Comment (lwander) [inline builtins.go]: I like it

Comment (sparkprime) [reply builtins.go]: these kinds of abstractions are basically mandatory given the error boilerplate

**Pull request merged**